src/utils/DatasetsUtils.js

```diff
@@ -61,10 +61,21 @@ const getAllChildrenDatasetsNames = (initialDatasetId, datasets) => {
   return datasetTree;
 };
 
+const removeUndefinedValuesBeforeCreatingDataset = (values) => {
+  if (!values) return;
```

Comment:
Sure there are any chance that `values` can be equal to _false/true_ or _0_ ?

Reply:
It can't be equal to _false/true_ or _0_ because RHF returns an object with `getValues()` function, this check is more about avoiding an unexpected `undefined` or `null` :slightly_smiling_face: